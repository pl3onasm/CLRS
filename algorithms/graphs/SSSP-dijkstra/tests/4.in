0

0 1 8
0 2 2
0 3 5
3 1 2
2 3 2
2 4 5
1 5 13
3 4 1
3 5 6
6 3 3
6 4 1
5 6 2
7 5 3
6 7 6
