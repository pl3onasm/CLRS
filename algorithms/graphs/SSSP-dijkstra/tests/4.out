
--------------------
 Graph
 Directed
 Weighted
 Vertices: 8
 Edges: 14
--------------------
  0[3]: 3 (5), 2 (2), 1 (8)
  1[1]: 5 (13)
  2[2]: 4 (5), 3 (2)
  3[3]: 5 (6), 4 (1), 1 (2)
  4[0]
  5[1]: 6 (2)
  6[3]: 7 (6), 4 (1), 3 (3)
  7[1]: 5 (3)
--------------------


Shortest paths
Source: 0
---------------------------------
Vertex: Parent, Distance from src
---------------------------------
  1: 3, 6.00
  5: 3, 10.00
  0: NIL, 0.00
  4: 3, 5.00
  3: 2, 4.00
  7: 6, 18.00
  2: 0, 2.00
  6: 5, 12.00
---------------------------------
